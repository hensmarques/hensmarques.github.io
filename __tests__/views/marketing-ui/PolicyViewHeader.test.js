import React from 'react';
import PolicyViewHeader from '../../../src/views/marketing-ui/PolicyViewHeader';

describe('<PolicyViewHeader/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<PolicyViewHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
