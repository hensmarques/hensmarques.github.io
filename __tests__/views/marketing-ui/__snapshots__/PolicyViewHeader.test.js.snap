// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PolicyViewHeader/> renders and matches the snapshot 1`] = `
<nav
  className="alt-nav"
>
  <div
    className="nav-container"
  >
    <Link
      className="logo"
      replace={false}
      to="/home"
    >
      <img
        className="logo-img"
        src="images/logos/company-logo-icon.png"
      />
    </Link>
    <SiteMapNav />
  </div>
</nav>
`;
